auto-check-fix-commit/git-bisect-runner.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+for arg in "$@"; do
+  shift
+  case "$arg" in
+  '--flaky') set -- "$@" '-f' ;;
+  '--fixed') set -- "$@" '-x' ;;
+  *) set -- "$@" "$arg" ;;
+  esac
+done
+
+OPTIND=1
+while getopts "f:x:" opt; do
+  case "$opt" in
+  'f') falky_commit=$OPTARG ;;
```

Comment:
Correct misspelled `falky` (here and once more below). I'd not mind some misspelling, but this one is about the main concept.

Reply:
Corrected the spelling.